nni/compression/pytorch/speedup/jit_translate.py

```diff
@@ -763,7 +767,8 @@ def hook_to_dtype_layout(positional, keyword, undetermined, undetermined_special
 
     def ret_func(*args):
         the_self = args[0]
-        the_self = layout_names_to_trans_dict[to_layout](the_self)
+        if the_self.layout != to_layout:
```

Comment:
is this because pytorch 1.10 do not have `to_dense`? what about pytorch 1.9? better to add a comment here.

Reply:
it's not support to do transform from torch.strided to torch.strided by to_dense on cuda backend. actually it's not necessary to do transform when source equals to aim.

Reply:
and it's support to do transform from torch.sparse_coo to torch.strided by to_dense on both cpu or cuda backend.